src/views/registration/attendees/fields.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * This template renders a the fields for a ticket
+ *
+ * @version TBD
+ *
+ */
+
+$ticket = $this->get( 'ticket' );
+
+$meta   = Tribe__Tickets_Plus__Main::instance()->meta();
```

Comment:
This logic could be moved to the method that is including this template so that the `$fields` variable is received ready to use and not computed here.

Reply:
Use the `tribe` service locator here: the `instance` method of `Tribe__Tickets_Plus__Main` code is:
```php
public static function instance() {
	return tribe( 'tickets-plus.main' );
}
```

So just use `tribe( 'tickets-plus.main' );`